src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ColorSupplier color = new ColorSupplier();
+        ball.setNumber(random.nextInt(100));
+        ball.setColor(color.getRandomColor());
+        return ball.toString();
```

Comment:
return ball itself, let consumer trasform it to string if needed, our solution should allow more options than just string representation